### ISSUE TITLE
Brave hangs on this site

### ISSUE THREAD

**Issue by angelog0** [brave/brave-browser#17617]
After typing the credentials [here](https://my.acea.it/myacea/it/signin), Brave hangs, a wheel shows up rotating indefinitely. Doing the same things, in parallel, with Chromium, works. It works if I disable Brave for that site.

I have seen this on GNU/Linux Mint 20.2 with

- Brave 1.28.105
- Chromium  92.0.4515.131 

All 64 bit, x86_64.

**Comment by ryanbr** [comment]
Hey @angelog0  Just tested the site,  Does it work in private window mode (no extensions enabled)? 

My testing so far, clearing cookies between each.

https://gif.fanboy.co.nz/gif/acea.gif

**Comment by angelog0** [comment]
No, @ryanbr, i does not work. I have to disable Brave also in private mode. I do not have extensions...
